### ISSUE TITLE
yarn build has not generate 'build/node_modules' .....

### ISSUE THREAD

**Issue by blanklong28** [reactjs/reactjs.org#3657]
accroding to the document [development-workflow](https://zh-hans.reactjs.org/docs/how-to-contribute.html#development-workflow), I ran the command `yarn build react/index,react/jsx,react-dom/index,scheduler --type=NODE` in my react-clone project，then it hasn't show me the document's respects（generate `build/node_modules/react` `build/node_modules/react-dom` folders）。The following picture is the command actually created...
![image](https://user-images.githubusercontent.com/13881250/116020286-93269e00-a678-11eb-92e3-b7ecbfb9c272.png) 

Is there someone can help me to use `yarn link` to point `react` `react-dom` to my local build folder???

**Comment by eps1lon** [comment]
Thanks for the report.

With the recent state of `master` the following folder structure is created for me when running `yarn build react/index,react/jsx,react-dom/index,scheduler --type=NODE`:
![Screenshot from 2021-04-26 11-35-22](https://user-images.githubusercontent.com/12292047/116062790-84de7f00-a684-11eb-8c56-f18f8c69cf74.png)

```bash
.
├── build
│   ├── facebook-www
│   │   └── shims
│   ├── node_modules
│   │   ├── jest-react
│   │   │   └── cjs
│   │   ├── react
│   │   │   └── cjs
│   │   ├── react-dom
│   │   │   └── cjs
│   │   └── scheduler
│   │       ├── cjs
│   │       └── umd
│   └── react-native
│       └── shims
```

Notice `build/node_modules` where the built packages are located.

A couple of things you can debug:
1. Check if your editor excludes `node_modules`
    In VSCode that would be `files.exclude`. Or use a different tool to print directories e.g. `ls -la build`. Some repositories have node_modules hidden e.g. https://github.com/facebook/jest/blob/64d5983d20a628d68644a3a4cd0f510dc304805a/.vscode/settings.json#L5-L6
1. which version of the repository have you checked out?
    You can run `git status` and post the result.
1. What operating system are you using (some linux distribution, windows, macOS etc)?
    You can run `npx envinfo` and post the result.

**Comment by blanklong28** [comment]
Base your solutions，I haven't solve it......
And the following pictures are my settings or project folder list~~~
![image](https://user-images.githubusercontent.com/13881250/116174118-5faf4680-a740-11eb-9aea-106df8df6fb3.png)
![image](https://user-images.githubusercontent.com/13881250/116174473-127fa480-a741-11eb-98f2-0f5de76e3c5a.png)
![image](https://user-images.githubusercontent.com/13881250/116174577-42c74300-a741-11eb-9067-575a4b0598ae.png)

**Comment by blanklong28** [comment]
@eps1lon  This morning, I delete the build folder, re-ran the command `yarn build react/index,react/jsx,react-dom/index,scheduler --type=NODE`，And I checked folder structure in my windows file manager, but still no node_modules in the build folder

**Comment by eps1lon** [comment]
Looks like you made some changes to `package.json` and `yarn.lock`. Does the issue still occur in a completely new clone without any changes applied?

**Comment by blanklong28** [comment]
> Looks like you made some changes to `package.json` and `yarn.lock`. Does the issue still occur in a completely new clone without any changes applied?

yes. I cloned a new one，did not modify any files including `package.json` and `yarn.lock`。When I run `yarn build react/index,react-dom/index,scheduler --type=NODE` ,  got the same results。
